### DISCUSSION TITLE
Antarctic sea ice

### DISCUSSION THREAD

**Discussion by dabail10** [NCAR/cesm_dev#12] [2. Model bias]
The Antarctic sea ice extent is too large in winter. This is for two historical runs (092 and 098b).

![8ba8d1c0c0552a059ece1d9278c26c17c9379f559c6a70312a5b30b44ff5becc](https://github.com/user-attachments/assets/f8a3920f-f8e7-4570-b859-57edb25db19f)

**Comment by duvivier** [comment]
This bias has continued into more of the test runs. It's present in PI and historical runs and current tests (e.g. #40  or #45) modifying properties have not impacted it substantially and we do not believe it will be something we can fix with sea ice albedo tuning. There has been some seasonality shift in recent tests that we also do not understand well (e.g. #48 & #49). 

PCWG winter meeting discussed possibilities about what could be causing this showstopper bias. Slides about our investigation can be found [here](https://docs.google.com/presentation/d/1oDc9RhMa-d28XOsHO94OZ8UYqd6pUGCtYnPkeqOemwI/edit?usp=sharing).

<img width="1256" alt="Screenshot 2025-03-07 at 8 35 45 AM" src="https://github.com/user-attachments/assets/7c6b2431-accb-442a-8cba-cbe437e8fe41" />

The CESM3 ***WINTER*** sea ice is the problem, while summer looks okay. But winter ice area is way outside historical bounds as well as both the CESM1-LE and CESM2-LE. There is also no paleo evidence that we're aware of to suggest the ice has ever been this extensive. 

Other WG folks joined the discussion and we came up with the following ideas to test:
1. Do some model archeology to see when the bias started. I will do this with @dabail10 and look at old test simulations.
2. @dabail10 and I will check if this is primarily an ice edge issue or if it's also related to too high of concentrations within the pack. We know there is an ice edge bias, but we'll quantify this better.
3. Run a nudged CESM3 simulation to see if atmospheric SH jet location or strength biases might impact the sea ice. There were other reasons to run this and @islasimpson and @JulioTBacmeister are working on getting this set up.
4. Investigate if extensive (liquid?) cloud or precipitation biases exist in CAM. We've spoken with @adamrher  about this and will look further.
5. Check if teleconnections of the SAM or ENSO might be impacting the bias here. There were some PCWG members who volunteered to help do some analysis on this.
6. Check if other MOM6 groups have cold biases in this area. @iangrooms and @gustavo-marques have been emailed and Ian suggested that tuning the GM parameter might help substantially.  Ian pointed out that in Adcroft et. al 2019 (http://dx.doi.org/10.1029/2019MS001726) they showed that in MOM6 1/2deg there are biases in winter (September) similar to ours when GM is turned on (panel H) compared to when it's turned off (panel I). Ian is going to run some CESM3 tests where he tunes the GM parameter since he thinks that likely needs some adjustment. It does look like the GM tuning affects summer sea ice less (see bottom row), so hopefully this could improve winter ice without degrading summer ice too much.
<img width="738" alt="Screenshot 2025-03-07 at 8 19 28 AM" src="https://github.com/user-attachments/assets/6d6cdd45-497e-40d7-b509-f1c316330e88" />

**Reply by duvivier** [reply]
We did some digging and found the bias starts around run 91 and 92 when some changes to the ocean vertical coordinate were made.

See below from CVDP:
![image](https://github.com/user-attachments/assets/0dc790d3-6def-46a1-8687-dedd66b869ea)

See here the seasonal cycles for 78 (the last test we could find data for) and 91 and 92:
![image](https://github.com/user-attachments/assets/6a629a74-93b3-4dfb-bd40-c8730d2a9b0b)
![image](https://github.com/user-attachments/assets/54fd35a3-5488-48a8-923a-86c123fc0b68)

**Reply by adamrher** [reply]
92 is when we switched from z* to the HYCOM1 vertical coordinate. IIRC, the ocean folks quite like the HYCOM1 coordinate...

**Reply by iangrooms** [reply]
We tried reducing the GM coefficient and it had the opposite effect from what's in the OM4 paper. It's possible that the response in the OM4 paper is nonlinear -- going from GM on to GM off is a big enough change that the response may not be representative of what happens when you just reduce GM. It's also possible that their model just responds differently than ours.

In the Weddell (and Ross?) sea the temperature is negatively stratified: cold at the surface and warmer beneath. GM restratifies, so it pushes the warm water down. In other simulations we had problems with big, unrealistic polynyas opening up in the Weddell Sea, and increasing GM fixed that. So in that case (increasing GM) => (more ice). We tried reducing GM to reduce ice in recent coupled simulations and saw the opposite effect: reducing GM increased ice, exacerbating the bias. The bias is not in the Weddell sea though, so the mechanism is not necessarily wrong, just probably not relevant.

Since the sea ice bias _is_ sensitive to GM, we can still try using GM to reduce the bias, but we evidently have to tune in the opposite direction. GM restratifies by pushing light water over dense water. Near the surface in the ACC the lighter water on the north side tends to also be warmer, so in that region GM's restratifying action will push warm water south.

**Reply by duvivier** [reply]
@iangrooms Thanks for explaining all the testing here. It is interesting we're seeing opposite results with reducing GM and then getting increased ice. It sounds like maybe increasing GM might improve the sea ice bias by pushing the warmer, light water southward? Fingers crossed this could help!

**Reply by dabail10** [reply]
I guess I had this backwards. I was thinking of the isopycnals flattening or steepening. So, I had naively assumed that steepening them would lead to more circumpolar deep water coming up. However, having more warm water moving poleward at the surface is probably better.

**Comment by islasimpson** [comment]
@cecilehannay To start a nudging run, I'm wondering what tag it would be best to use?  Perhaps there's a BLT1850 simulation of yours that I could clone?

**Reply by cecilehannay** [reply]
I would use alpha06b. 
130 is our new baseline with alpha06b. 
We are also running 133 starting with a more spinup land

**Reply by islasimpson** [reply]
OK, thanks.  I'll clone 130.

**Reply by islasimpson** [reply]
Right now I'm testing with a clone of 130.  I'm initializing with the restarts from year 21 (before the lab sea freeze) of 130 since the original initial conditions that were used for 130 are no longer available.  But I'm wondering whether we'd actually be wanting to start from a G-case i.e., start from something that has better sea ice and see if it gets worse as opposed to start from bad sea ice and see if it gets better.  Any thoughts on how to initialize the nudging run? @dabail10 @duvivier @cecilehannay @gustavo-marques

**Comment by duvivier** [comment]
I guess I'm inclined to start from a coupled simulation rather than a G-case. We know there's a lot of adjustment with the G-case initialization in general and so I think we should try nudging and see if that helps with an otherwise "balanced" fully coupled case in terms of the winds. Or at least if they lead to ice trending in the right direction.

**Reply by islasimpson** [reply]
OK, sounds good.  Then I can just proceed with what I'm currently using.

**Comment by islasimpson** [comment]
Here's a look at the lower tropospheric zonal wind biases in the nudged run, the regular 130, and CESM2.  The winds over the Southern Ocean were too strong in CESM2.  In CESM3, the winds are shifted equatorward and are now weaker around Antarctica.

<img width="1368" alt="Screenshot 2025-03-26 at 4 36 45 PM" src="https://github.com/user-attachments/assets/a6bb9182-ca45-4c20-a7ed-ca7c23f8f89b" />

<img width="1363" alt="Screenshot 2025-03-26 at 4 37 39 PM" src="https://github.com/user-attachments/assets/e1b0900b-8742-49a8-95f0-ff75068fe03d" />

The zonal means below make it clear that the strength of the westerlies is much improved in 130 compared to CESM2.  But they're too far equatorward.  CESM used to be kind of an exception in having the location of the westerlies in the right place.  Being too far equatorward is a common issue among climate models.  But it seems like something we should maybe be trying to improve given the implications for sea ice.  Overall, I think we're not necessarily more biased than we were in CESM2 but we now have a different bias (better jet strength, worse jet position).

<img width="1188" alt="Screenshot 2025-03-26 at 5 34 50 PM" src="https://github.com/user-attachments/assets/f76a84ff-d904-48f2-b88b-3c56ddf13d32" />

**Reply by duvivier** [reply]
@islasimpson These are very helpful, thank you. I hadn't realized just how strong the jet was in CESM2, but I agree now we're about the right strength but too far north. Do you think that in 130 the equatorward bias is somehow partially driven by the sea ice being too extensive and so too far north? Like the baroclinicity is highest northward due to the ice edge and this contributes to the problem?

**Comment by islasimpson** [comment]
I think it's not totally straightforward to conclude from the biases shown above what we need to be doing to fix the sea ice problem and whether it's an atmosphere problem, an ocean problem or a coupled problem.

Here's a plot that's showing the zonal wind biases in DJF in the 130 BLT1850 case (used above) and then in the recent FLTHIST case that @cecilehannay  ran from 2000-2009.  While the FLTHIST case is a short run, the zonal wind biases relative to ERA5 are totally different in the FLTHIST case compared to the BLT1850 case.  

![Screenshot from 2025-03-27 20-58-09](https://github.com/user-attachments/assets/a541da5f-621f-40a4-be3a-c81cdbc38127)

Since the FLTHIST case above is a short run, we can go back to an older generation, the 121 series and do the same comparison, and we see a similar thing:

![Screenshot from 2025-03-27 21-02-23](https://github.com/user-attachments/assets/66e7f22b-33ed-436e-a9f1-893b32a1650b)

So, it's clear that the jet stream biases we have in a coupled 1850 case are not the same as the jet stream biases we get from the atmosphere alone when given prescribed observation-based SSTs.

Part of this difference may be because of the different time periods.  Here is a comparison of the 121 BHIST case in 2000-2009 which is the period used for the AMIP runs with the 1850 case.  The biases in the winds in the Southern Ocean are reduced probably because the ozone whole and CO2 are driving a poleward shift of the westerlies.    But the biases aren't gone.

![Screenshot from 2025-03-27 21-18-16](https://github.com/user-attachments/assets/10e7039e-68b5-47aa-95f3-1f93eecc3b1a)

I'm not sure where this leaves us.   I think it suggests (a) that the biases in the zonal wind relative to ERA5 are a coupled issue and not an atmosphere alone issue and (b) our biases are likely smaller than we would infer from looking at BLT1850 because the winds are changing over time.

Here's the BLT1850 vs FLTHIST (2000-2009) comparison for the other seasons.

MAM:
![Screenshot from 2025-03-27 21-20-14](https://github.com/user-attachments/assets/43bb1398-3c05-4d06-ba23-64e8f5ac00d6)
![Screenshot from 2025-03-27 21-20-33](https://github.com/user-attachments/assets/1550101d-7097-40e9-b2f4-bf9e43ce1a7d)

JJA:
![Screenshot from 2025-03-27 21-22-21](https://github.com/user-attachments/assets/68d8f6b4-8acd-401b-9359-5942879ec754)
![Screenshot from 2025-03-27 21-23-03](https://github.com/user-attachments/assets/b5d92353-18b7-441d-a54e-63bd7d5a2810)

SON:
![Screenshot from 2025-03-27 21-25-37](https://github.com/user-attachments/assets/85f16622-5e7d-4bd8-9ebd-615756d7036e)
![Screenshot from 2025-03-27 21-25-52](https://github.com/user-attachments/assets/59897e8e-475f-4481-86ee-0bff31510e0e)

**Reply by duvivier** [reply]
@islasimpson  I personally think this is probably a coupled problem, not just the jet. I can't imagine "fixing" a bias like this is easy in the atmosphere alone if that was the main cause. But I wonder if the sea ice is contributing to the problem (See my previous comment). If you see different biases in the F-case, that suggests to me it's definitely a coupled issue. I do think we should continue some tests with the GM tuning that @iangrooms and @gustavo-marques suggested to see if that can help with the ice as well and then we can see if that has an impact on the jet location.

**Comment by islasimpson** [comment]
A suggestion from @adamrher was to check whether the zonal wind biases changed a lot between 091 and 092 (note the titles in the plots are wrong and say (191 and 192) when the ocean vertical coordinate changed.  There does not seem to be a big difference between these two runs.

<img width="778" alt="Screenshot 2025-03-28 at 5 57 07 PM" src="https://github.com/user-attachments/assets/722560ff-3da6-46b2-a856-96bc1a8f70c8" />
<img width="759" alt="Screenshot 2025-03-28 at 5 57 34 PM" src="https://github.com/user-attachments/assets/f175b237-2157-4d73-98b3-ebbd3bf96020" />
